config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  devise_for :users, controllers: { registrations: 'users/registrations',
+                                    sessions: 'users/sessions' }
+  resources :users, only: [:show, :update]
```

Comment:
switch update to a collection route, read [this](http://guides.rubyonrails.org/routing.html#adding-more-restful-actions)